.github/workflows/deploy.yaml

```diff
@@ -5,8 +5,18 @@ on:
     tags: [v*]
   workflow_dispatch:
     inputs:
-      features:
-        description: 'Optional cargo features (currently supported: "mimalloc-allocator")'
+      predefined_feature:
+        description: 'Select a predefined feature'
+        type: choice
+        options:
+          - 'None'
+          - 'mimalloc-allocator'
+          - 'tokio-console'
+        default: 'None'
+        required: false
+      custom_features:
+        description: 'Additional custom features (space-separated, e.g., "feature1 feature2")'
+        type: string
```

Comment:
Can anyone run this? This thingy here is prime for shell injection and gh only goes so far as to censor the secrets but you can always do something like `sed 's/./& /g'` and add a space between chars to circumvent that